scripts/util/keystore-cert-import

```diff
@@ -121,6 +121,10 @@ if [ -f "$LIBVIRTD_FILE" ]; then
     ln -sf /etc/pki/libvirt/servercert.pem /etc/pki/libvirt-vnc/server-cert.pem
     ln -sf /etc/pki/libvirt/private/serverkey.pem /etc/pki/libvirt-vnc/server-key.pem
     cloudstack-setup-agent -s > /dev/null
+
+    QEMU_GROUP=$(cat /etc/group | awk -F: '{ print $1}' | grep qemu)
+    chgrp $QEMU_GROUP /etc/pki/libvirt /etc/pki/libvirt-vnc /etc/pki/CA /etc/pki/libvirt/private /etc/pki/libvirt/servercert.pem /etc/pki/libvirt/private/serverkey.pem /etc/pki/CA/cacert.pem /etc/pki/libvirt-vnc/ca-cert.pem /etc/pki/libvirt-vnc/server-cert.pem /etc/pki/libvirt-vnc/server-key.pem
```

Comment:
@nvazquez 
what if there are multiple groups in the output ?

Reply:
@nvazquez 
since the user/group is specified in `/etc/libvirt/qemu.conf`, should the group be fetched from qemu.conf ?

Reply:
That's a good idea @weizhouapache, thanks. I've created a documentation PR with a note for the limitation you spotted: https://github.com/apache/cloudstack-documentation/pull/306 but I think using the qemu.conf is a better approach cc @borisstoyanov

Reply:
@nvazquez 
1. You can add 'head -n1' to get the first result.
2. You may need to check if result is empty or not

Reply:
Thanks @weizhouapache, I've used tail actually to get the last value